pnpm-lock.yaml

```diff
@@ -4390,7 +4396,7 @@ packages:
     peerDependencies:
       graphql: ^0.8.0 || ^0.9.0 || ^0.10.0 || ^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0
     dependencies:
-      '@graphql-codegen/plugin-helpers': 2.4.2_graphql@15.8.0
```

Comment:
should these @graphql-codegen/plugin-helpers updates be in this pr?

Reply:
Im not sure how that made its way in here. Ill revert the entire pnpm-lock file and try again.